README.md

```diff
@@ -62,6 +46,8 @@ Allows to wrap an error (either std error or errkit error) with higher level mes
 
 ### errors.Is and errors.As
 Allows to use standard errors matching
+
+`Errkit` also has an aliases `errkit.Is` and `errkit.As`
```

Comment:
```suggestion
`Errkit` also has aliases `errkit.Is` and `errkit.As`
```

Reply:
Thanks, I did text checking, now they seems to be written in a better way.